aws/lambda/opensearch-gha-jobs/Makefile

```diff
@@ -0,0 +1,11 @@
+prepare: clean
+	mkdir -p ./packages
+	pip install --target ./packages -r requirements.txt
+	cd packages && zip -r ../opensearch-gha-jobs-deployment.zip .
+	zip -g opensearch-gha-jobs-deployment.zip lambda_function.py
+
+deploy: prepare
+	aws lambda update-function-code --function-name opensearch-gha-jobs --zip-file fileb://opensearch-gha-jobs-deployment.zip
```

Comment:
can it be assumed that the aws cli will always be there? (asking b/c I see you install and remove pip packages)

Reply:
The cli is installed at the system level, so yeah, it would always be there.  The pip packages directory is to uploaded to the lambda (as a zip), so it only has dependencies as listed in `requirements.txt`